cityofaustin/atd-moped
# Add a combined point + line AGOL layer

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/14477


## Testing
**URL to test:** Local

**Steps to test:**
1. Start moped with a fresh copy of production data
2. Follow the steps to run the ETL: https://github.com/cityofaustin/atd-moped/pull/1148#issue-1927133588

Check out the results in the [AGOL dataset](https://austin.maps.arcgis.com/home/item.html?id=1c084c8756a84e6db7e2796c98c850a2), which now has a 3rd layer for the combined features.

<img width="362" alt="Screenshot 2023-11-03 at 12 36 50 PM" src="https://github.com/cityofaustin/atd-moped/assets/14793120/57059f33-6a5e-4b8d-a9be-ae43fbf447ab">


---
#### Ship list
